test/blackbox-tests/test-cases/sendfile-large-file.t

```diff
@@ -0,0 +1,22 @@
+We create a large file and check that it is copied completely.
+
+  $ cat > dune-project << EOF
+  > (lang dune 1.0)
+  > EOF
+
+  $ cat > create.ml << EOF
+  > let () = Unix.truncate "file.dat" 0x1_00_00_00_03
```

Comment:
Can we make sure somehow that the file that is being created here is getting created on a file system that supports sparse files?

Looking at [this comparison list](https://en.wikipedia.org/wiki/Comparison_of_file_systems) it seems like it should work on Windows assuming its running on NTFS and not some FAT-varaiant and it works on MacOS as long as the FS is APFS and not HFS+ (BSDs and Linux seem to be pretty well-covered wrt to sparse file FS support).

Reply:
The test is disabled by default because it's pretty slow in its current implementation. If we can make it use sparse files in the usual test targets, I think we can enable it by default. In any case I'll add a note about that. If it's complicated to use sparse files I'll disable the test.